package.json

```diff
@@ -44,7 +41,7 @@
   },
   "lint-staged": {
     "**/*.{js,jsx,ts,tsx,json}": [
-      "prettier --plugin prettier-plugin-organize-imports --write"
+      "pnpm format -- --linter-enabled=false --no-errors-on-unmatched"
```

Comment:
See [Biome docs regarding lint-staged](https://biomejs.dev/recipes/git-hooks/#lint-staged) for suggested commands to run here.

Note that when I first tried to push a commit, lint-staged will run the above command successfully, but would not add any changes to the commit I was trying to push. It left the changes saved and waiting to be staged for another commit.

Reply:
That's not nice, but I believe there is an option to fix that